### ISSUE TITLE
Removing filter pill from search input raises aria-hidden / tabindex violation

### ISSUE THREAD

**Issue by jacobtylerwalls** [archesproject/arches#11240]
- Search > Apply a resource type filter
- Clear the filter

Result:
```html
search:1 Blocked aria-hidden on a <span> element because the element
that just received focus must not be hidden from assistive technology users.
Avoid using aria-hidden on a focused element or its ancestor. Consider using
the inert attribute instead, which will also prevent focus. For more details,
see the aria-hidden section of the WAI-ARIA specification at
https://w3c.github.io/aria/#aria-hidden.
<span class=​"select2-selection__choice__remove" role=​"presentation" aria-hidden=​"true" tabindex=​"0" aria-label=​"remove item">​×​</span>​
:8000/search?paging-filter=1&tiles=true&format=tilecsv&reportlink=false&precision=6&total=9&exportsystemvalues=false&provisional-filter=%5B%7B%22provisionaltype%22%3A%22Provisional%22%2C%22inverted%22%3Afalse%7D%5D:1 Blocked aria-hidden on a <span> element because the element that just received focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden. <span class=​"select2-selection__choice__remove" role=​"presentation" aria-hidden=​"true" tabindex=​"0" aria-label=​"remove item">​×​</span>​
```
